.travis.yml

```diff
@@ -16,7 +16,7 @@ before_script:
         if [ "${TRAVIS_EVENT_TYPE}" = "cron" ]; then
             PYTEST_ADDOPTS="-m 1"
         else
-            PYTEST_ADDOPTS="-m 'not optional'"
+            PYTEST_ADDOPTS="-m 'normal and functional'"
```

Comment:
You have to use `or` instead of `and`, because the tests categories are distinct now.

Reply:
OK thanks for mentioning it. I did not understand it.